modules/@apostrophecms/template/index.js

```diff
@@ -327,8 +328,12 @@ module.exports = {
         });
         // Final class should win
         dirs.reverse();
-        if (self.options.viewsFolderFallback) {
-          dirs.push(self.options.viewsFolderFallback);
+
+        const viewsFolderFallback = self.options.viewsFolderFallback ||
+          path.join(self.apos.rootDir, 'views');
+
+        if (viewsFolderFallback) {
```

Comment:
This will always be truthy, based on line 332. May as well make clear it can't be entirely shut off by dropping the `if`.